analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -90,7 +90,7 @@ def add_arguments_to_parser(parser):
                                 "Please consult the User guide on how a "
                                 "Skipfile should be laid out.")
 
-    skip_mode.add_argument('--files',
+    skip_mode.add_argument('--file',
```

Comment:
Please add some valid examples here, to the check command and to the user guide: `"main.cpp"`, `"lib/*.cpp"`, `"*/test*"`